MagickCore/resample.h

```diff
@@ -57,6 +57,7 @@ typedef enum
   LanczosSharpFilter,
   Lanczos2Filter,
   Lanczos2SharpFilter,
+  MagicSharp2013Filter,
```

Comment:
New enum values should be added at the end to maintain backwards compatibility

Reply:
Done